lib/npg_qc/autoqc/checks/genotype.pm

```diff
@@ -658,7 +661,7 @@ sub _set_attrib_by_geno_refset {
   my $name   = fileparse($self->geno_refset);
 
   $self->_set_sequenom_plex($name);
-  $self->_set_snp_call_set($name);
+  $self->_set_ssnp_call_setnp_call_set($name);
```

Comment:
This invocation doesn't look right, and doesn't turn anything up in devel code.

Reply:
This is an accidental edit, the code is now reverted back to what it was. The fact that the tests still pass point to low test coverage. Thank you very much for spotting this error.